tsdb/head_read_test.go

```diff
@@ -129,18 +129,7 @@ func TestBoundedChunk(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(fmt.Sprintf("name=%s", tc.name), func(t *testing.T) {
-			chunk := boundedChunk{tc.inputChunk, tc.inputMinT, tc.inputMaxT}
-
-			// Testing Bytes()
-			expChunk := chunkenc.NewXORChunk()
-			if tc.inputChunk.NumSamples() > 0 {
-				app, err := expChunk.Appender()
-				require.NoError(t, err)
-				for ts := tc.inputMinT; ts <= tc.inputMaxT; ts++ {
-					app.Append(ts, float64(ts))
-				}
-			}
-			require.Equal(t, expChunk.Bytes(), chunk.Bytes())
+			chunk := boundedIterable{tc.inputChunk, tc.inputMinT, tc.inputMaxT}
```

Comment:
nit s/chunk/iterable

Reply:
Updated in https://github.com/prometheus/prometheus/pull/13123/commits/c004db1558f20f6f8824529b712764c398f13bad